pages/user guides/discover_nmap.md

```diff
@@ -0,0 +1,82 @@
+last_updated: September 2020
+summary: "How to use nmap to scan your network"
+sidebar: main_sidebar
+permalink: discover-nmap.html
+---
+
+
+
+As part of your cloud migration journey it is important to have all the tools at your disposal. In addition to our proprietary tools, you can now use [Nmap](https://nmap.org/) to facilitate your discovery process. 
+The way it works is very simple and with the full power of nmap available to you, discovery of even those hard to reach places is possible. 
+ 
+Nmap has developed some powerful capabilities and good [documentation](https://nmap.org/book/host-discovery-find-ips.html) that will allow you to identify  all the hosts you have running on your network, the ports and services that they have exposed and even some vulnerabilities discovered by the open source community. 
+ 
+Nmap is capable of producing its output in an XML file, allowing you to inspect the raw scan output before sending it to the Tidal Migrations API with our CLI,  [Tidal Tools](https://get.tidal.sh). Once uploaded to the Tidal Migrations Platform, you will be able to visualize your network devices, track your complete server inventory and build on this data with other discovery methods, allowing you to make informed decisions on your cloud migration path.
+
+
+## Using Nmap with Tidal Tools
+ 
+By leveraging the power of **Tidal Tools** you can send the output generated by Nmap to Tidal migration API.
+ 
+To get the process started your must follow these steps first:
+1. Install [Tidal Tools](https://guides.tidalmg.com/tidal-tools.html)
+2. [Login to Tidal Tools](https://guides.tidalmg.com/tidal-tools.html#login) with **tidal login**
+3. Run Nmap [how to install](#installing-nmap) with the flags of your choosing and save the output on an XML file. For example, 
+```
+sudo nmap -sS -p80,443,8080,8443,1433,1521,27017 <ip-address> -oX my-network.xml
+```
+Note: the **-sS** flag will enable TCP SYN port scan and the **-oX** specifies the output as a XML file.
+[Want more scanning options?](#nmap-usage) 
+ 
+4. Run this Tidal Tools command to upload your previously generated Nmap output to your Tidal Migrations account 
+```
+tidal sync nmap <filename.XML> 
+``` 
+ 
+5. Head out to your Tidal account!
+ 
+
+
+## Installing Nmap
+
+Nmap ("Network Mapper") is a free and open source utility for network discovery and security auditing. Many systems and network administrators also find it useful for tasks such as network inventory, managing service upgrade schedules, and monitoring host or service uptime.
+ 
+You can find all the documentation in its official site [Nmap](https://nmap.org/) and instructions for how to download based on your environment [download](https://nmap.org/download.html)
+
+
+## Nmap usage
+
+Nmap offers a wide range of utilities and commands, such as Port scanning, Host discovery, Service and version detection to name a few. Here some basic examples for how to do some nmap scanning.
+ 
```

Comment:
Consider a note here listing the features or attributes that Tidal currently captures. Something like

```
Note: Currently, Tidal Migrations supports the collection of Hosts (IP addresses), PTR records, their open ports, the ports status, the port protocol (TCP/UDP), and the services running in the port, including the version.
```

Thinking so it's clear when someone provides an Nmap file with _more_ data and wondering why its not in the Platform after uploading. We might want to communicate this in other places. Maybe higher up on this page. Or in Tidal Tools itself as well, either in the usage text or the output shown to a user while or after uploading the data.